src/smpp_socket.erl

```diff
@@ -86,7 +90,7 @@
 % to handle all of them regardless.
 % Relative frequency of occurence of each one of them depends on
 % SMSC implementation and one's timings.
--type flow_control_reason() :: sending_expired | rate_limit | in_flight_limit.
+-type flow_control_reason() :: sending_expired | rate_limit | in_flight_limit | overload.
```

Comment:
wrong typespec, `overload` reason can't be returned in callback fun for `send_async`

Reply:
That's what you've already fixed by the last commit, right?

Reply:
fixed